tests/resources/config/config_global.json

```diff
@@ -0,0 +1,28 @@
+{
+  "host": "devx1.symphony.com",
+  "context": "context",
+  "port": 8443,
+  "pod": {
+    "host": "diff-pod.symphony.com",
+    "port": 8443
+  },
+  "agent": {
+    "host": "devx1.symphony.com",
+    "port": 443
+  },
+  "keyManager": {
+    "host": "devx1.symphony.com",
+    "context": "diff-context"
+  },
+  "sessionAuth": {
+    "scheme": "http"
+  },
+  "bot": {
+    "username": "tibot",
+    "privateKeyPath": "/Users/local/conf/agent/privatekey.pem"
```

Comment:
Python BDK should not support `privateKeyPath` (it's something already deprecated in Java), but `privateKey` > `path` instead.

Reply:
this was a mistake. it should not be used.